scss/_progress.scss

```diff
@@ -54,22 +54,22 @@
 //     .border-radius($border-radius);
 //     .box-shadow(inset 0 .1rem .1rem rgba(0,0,0,.1));
 //   }
-//   .progress[value]::-moz-progress-bar {
-//     background-color: #0074d9;
-//     border-top-left-radius: $border-radius;
-//     border-bottom-left-radius: $border-radius;
-//   }
-//   .progress[value="0"]::-moz-progress-bar {
-//     color: $gray-light;
-//     min-width: 2rem;
-//     background-color: transparent;
-//     background-image: none;
-//   }
-//   .progress[value="100"]::-moz-progress-bar {
-//     border-top-right-radius: $border-radius;
-//     border-bottom-right-radius: $border-radius;
-//   }
 // }
+.progress[value]::-moz-progress-bar {
+  background-color: #0074d9;
+  border-top-left-radius: $border-radius;
+  border-bottom-left-radius: $border-radius;
+}
+.progress[value="0"]::-moz-progress-bar {
+  color: $gray-light;
```

Comment:
Properties should be ordered min-width, color, background-color, background-image